src/file/file.rs

```diff
@@ -0,0 +1,282 @@
+use std::{fs, path::PathBuf};
+
+use super::{ArgumentToken, Expression, File, Line};
+
+pub fn read_ssh_config(path: &PathBuf) -> std::io::Result<File> {
```

Comment:
Consider documenting this method.
Is std::io::Result a good idea to include in public API?